doc/api/buffer.md

```diff
@@ -164,6 +164,15 @@ console.log(Buffer.from('fhqwhgads', 'ascii'));
 console.log(Buffer.from('fhqwhgads', 'utf16le'));
 // Prints: <Buffer 66 00 68 00 71 00 77 00 68 00 67 00 61 00 64 00 73 00>
 ```
+```js
```

Comment:
Welcome @juandi27 and thanks for the pull request! I think you've put the example code in the wrong place. I'm not seeing how it makes sense where it's been placed. That section is about character encodings. This example has nothing to do with that.